radixdlt/src/main/java/com/radixdlt/consensus/Proposal.java

```diff
@@ -79,8 +79,11 @@ private void setSerializerAuthor(byte[] author) throws CryptoException {
 
 	@Override
 	public String toString() {
-		return String.format("%s{author=%s view=%s}", getClass().getSimpleName(),
-			author.euid().toString().substring(0, 6), vertex.getView());
+		String who = author == null ? "null" : author.euid().toString().substring(0, 6);
```

Comment:
Why make null a string here (but not for the other properties)?

Reply:
The others aren't strings, so a string wouldn't work there, but an actual `null` would be fine here, so I'll change it to that.